docs/upgrading_wagtail_v2/index.rst

```diff
@@ -0,0 +1,158 @@
+==============================
+Upgrading to Wagtail Version 2
+==============================
+
+Wagtail 2.0 Requirements
+========================
+* Django versions: 1.11, 2.0 
+* Python versions: 3.4, 3.5, 3.6
+    - It's recommended that you upgrade your python version first to be compatible with python 3.4.
```

Comment:
What do you exactly mean? What is compatible with 3.4? Is python 3.6 compatible? Should I downgrade? Also you write 'first' so what is 'second'? Maybe:

Make sure your project runs on Python 3.4 or higher before upgrading Django and Wagtail.

Or do you mean something else?